src/Microsoft.ML.Data/Transforms/ConcatTransform.cs

```diff
@@ -527,6 +527,28 @@ private static VersionInfo GetVersionInfo()
 
         public override ISchema Schema => _bindings;
 
+        public ConcatTransform(IHostEnvironment env, IDataView input, string outputColumn, params string[] inputColumns)
+            : base(env, RegistrationName, input)
+        {
+            var cols = new Column[1];
+            cols[0] = new Column()
+            {
+                Name = outputColumn,
+                Source = inputColumns
+            };
+
+            var args = new Arguments()
+            {
+                Column = cols
+            };
+            Host.CheckValue(args, nameof(args));
+            Host.CheckUserArg(Utils.Size(args.Column) > 0, nameof(args.Column));
```

Comment:
I understand why we do it other constructor, because we get them as parameters, and you need to validate them, but here, you create args and it content specifically.
I would rather have validation of outputColumn, inputColumns, input, rather than copypaste validation from other constructor.

Reply:
make sense!